<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameEditor</class>
 <widget class="QDialog" name="FrameEditor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>623</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FrameEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListView {
    alternate-background-color: yellow;
    background-color: #697378;
	margin-right:10px
}

QListView {
    show-decoration-selected: 1; /* make the selection span the entire width of the view */
}

QListView::item:selected:active {
	 background-color: #96A4AD;
}

QListView::item:hover {
     background-color: #96A4AD;
}

 QHeaderView::section {
	 background-color: rgba(0,0,0,0%);
	 color: white;
	padding:5px;
text-transform:uppercase;
 }

QTableCornerButton::section {background-color: rgba(0,0,0,0); }

*::disabled
{
		background-color:#b4bdc6;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #475156, stop: 1 #74828B);
		color:black;
}

QMainWindow, QWidget{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #697378, stop: 1 #96A4AD);
}

*{
	font-size:14px;
	font-family:&quot;Bahnschrift&quot;;
	/*padding:5px;*/
	color:white;
}

*:diabled{
	    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #475156, stop: 1 #74828B);
}

QLabel,QRadioButton{
	background-color: rgba(0,0,0,0%)
}

QPushButton
{

    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #96A4AD, stop: 1 #697378 );

border: 2px solid #96A4AD;
border-radius:6px;

    min-width: 80px;
	min-height:30px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #74828B, stop: 1 #697378 );
    /*background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);*/
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: #535a66;
}

QPushButton:disabled {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #475156, stop: 1 #74828B);
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FFOECE, stop: 1 #FFFFFF);

}

QListView::item:selected:active {
	 background-color: #96A4AD;
}

QListView::item:hover {
     background-color: #96A4AD;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="5">
    <widget class="QGroupBox" name="gBConnections">
     <property name="title">
      <string>Frame objects Editor</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="treeViewFrames"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbAddNode">
            <property name="text">
             <string>Add Node</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbAddChild">
            <property name="text">
             <string>Add Child</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemoveObject">
            <property name="text">
             <string>Remove Object</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pbGraph">
     <property name="text">
      <string>Draw Frame Graph</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pbSave">
     <property name="text">
      <string>Save Frame Model</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pbLoad">
     <property name="text">
      <string>Load Frame Model</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="pbNewModel">
     <property name="text">
      <string>New Frame Model</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
